dev-python/pycryptodome/pycryptodomex-3.17.0.recipe

```diff
@@ -4,19 +4,20 @@ primitives.
 
 PyCryptodome is a fork of PyCrypto. It brings several enhancements with respect \
 to the last official version of PyCrypto (2.6.1).
-"
+
+This package ('pycryptodomex', mind the final "x") can co-exist with the 'pycrypto' \
+packages, as it installs as a Python module under the name 'Cryptodome'."
 HOMEPAGE="https://www.pycryptodome.org/"
-COPYRIGHT="2014-2022 Legrandin"
+COPYRIGHT="2014-2023 Legrandin"
 LICENSE="BSD (2-clause)
 	Public Domain
 	Unlicense
 	"
 REVISION="1"
-SOURCE_URI="https://pypi.python.org/packages/source/p/pycryptodome/pycryptodome-$portVersion.tar.gz"
-CHECKSUM_SHA256="0e45d2d852a66ecfb904f090c3f87dc0dfb89a499570abad8590f10d9cffb350"
+SOURCE_URI="https://github.com/Legrandin/pycryptodome/archive/refs/tags/v$portVersion.tar.gz"
```

Comment:
Is this the same as https://pypi.org/packages/source/p/pycryptodomex/pycryptodomex-$portVersion.zip ?

Reply:
If things aren't stable, then don't bother. The next update will probably break anyway.